JDRF/spirit
# DGST-8:  Set explicit size of hero svg wave for IE11

Ticket: 
https://jdrf.atlassian.net/browse/DGST-8

Style / output enhancements for IE11

To test:

1. Pull branch
2. In `doc-site` run `gulp`
3. View split screen hero wave (on right) in IE11 - should look correct (not super tiny)
